File: portefeuille-service/src/main/java/com/example/portefeuilleservice/Model/Portefeuil.java
package com.example.portefeuilleservice.Model;

import lombok.Data;

import javax.persistence.*;
import java.io.Serializable;

@Entity
@Data

public class Portefeuil implements Serializable {

    @Transient
    private String message;

    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String reference;
    private String proprietaire;
    private Double solde;

}

File: bankly_project_flutter/lib/Models/Portefeuil.dart
import 'dart:ffi';

import 'package:flutter/material.dart';

class Portefeuil {
  String message;

  int id;
  String reference;
  String proprietaire;
  double solde;

  Portefeuil(
      this.id, this.proprietaire, this.reference, this.solde, this.message);

  // factory Portefeuil.fromMap(Map portefeuilMap) {
  //   return Portefeuil(
  //        portefeuilMap['id'],
  //       portefeuilMap['proprietaire'],
  //        portefeuilMap['reference'],
  //       portefeuilMap['solde'],
  //        portefeuilMap['message']);
  // }
}
